### DISCUSSION TITLE
Test it first?

### DISCUSSION THREAD

**Discussion by cluttered** [openaudible/openaudible#1167] [General]
I have downloaded the latest version, couldn’t login via the web but managed cia the app itself. I can download a list of my titles and have downloaded one book.
If I try to convert then I get “pay up”!
So I am expected to lash out £20, not an inconsiderable sum, on the off chance it will work for me?
Is there seriously no way to test it first? Why not let me do one book, or the first 5 minutes of a book?
There also seems to be no direct support, just github. All seems a bit iffy.
If it works then I would happily buy it, but why cannot I try it first, that is usually a red flag.

**Comment by openaudible** [comment]
If it didn't work, you'd see a whole lot of unhappy people in our github issues. If you want a commercial quality app to back up your audible library, OpenAudible is the best there is. Lots of happy users out there. Our requests for refund is well under a half a percent.

**Comment by cluttered** [comment]
I decided rather than pay a large amount of money I would use ffmpeg and convert them all for free.

Works like a charm.

And way to go on support, only took a couple of months to get a response, very well done.

**Comment by oboe45** [comment]
Agreed, easily the best and very, very well supported.

On Sat, May 6, 2023 at 11:49 PM openaudible ***@***.***>
wrote:

> If it didn't work, you'd see a whole lot of unhappy people in our github
> issues. If you want a commercial quality app to back up your audible
> library, OpenAudible is the best there is. Lots of happy users out there.
> Our requests for refund is well under a half a percent.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/openaudible/openaudible/discussions/1167#discussioncomment-5826662>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APZFLTNUDILPM2FNPRLBLVTXE4LUHANCNFSM6AAAAAAWBPKK5Q>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***
> .com>
>
-- 
Bob Barker

**Comment by openaudible** [comment]
For some reason we don't get email notifications on discussions, so they sometimes go unanswered for long periods of time. Trying to find the setting or API request to monitor discussions.
